packages/core/contracts/insured-bridge/BridgePool.sol

```diff
@@ -385,6 +389,10 @@ contract BridgePool is Testable, BridgePoolInterface, ERC20, Lockable {
      *      quoteTimestamp. The OO acts to verify the correctness of this realized fee. Can not exceed 50%.
      */
     function relayDeposit(DepositData memory depositData, uint64 realizedLpFeePct) public nonReentrant() {
+        // If deposit quote time is before this contract's deployment, then clients will not be able to compute
+        // a realized LP fee %, so we should block such relays.
+        require(depositData.quoteTimestamp >= deploymentTimestamp, "Deposit before deployment");
```

Comment:
we could make this a modifier? guess it's less gas to have it this way though.

Reply:
Yea let's leave it like this for gas reasons. There are other require statements we could make modifiers but we chose to keep it inline for this same reason